4.2.0 of Social-core package breaks our tests

```
==================================== ERRORS ====================================
16
___ ERROR collecting ansible_catalog/common/tests/unit/test_keycloak_oidc.py ___
17
ImportError while importing test module '/home/runner/work/ansible-catalog/ansible-catalog/ansible_catalog/common/tests/unit/test_keycloak_oidc.py'.
18
Hint: make sure your test modules/packages have valid Python names.
19
Traceback:
20
/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/importlib/__init__.py:127: in import_module
21
    return _bootstrap._gcd_import(name[level:], package, level)
22
ansible_catalog/common/tests/unit/test_keycloak_oidc.py:4: in <module>
23
    from social_core.tests.backends.open_id_connect import OpenIdConnectTestMixin
24
E   ModuleNotFoundError: No module named 'social_core.tests.backends.open_id_connect'
```

--- a/ansible_catalog/common/tests/unit/test_keycloak_oidc.py
+++ b/ansible_catalog/common/tests/unit/test_keycloak_oidc.py
@@ -1,7 +1,7 @@
 import json
 
 from social_core.tests.backends.oauth import OAuth2Test
-from social_core.tests.backends.open_id_connect import OpenIdConnectTestMixin
+from social_core.tests.backends.test_open_id_connect import OpenIdConnectTestMixin
 
 
 class TestKeycloakOpenIdConnect(OpenIdConnectTestMixin, OAuth2Test):

